config/locales/en.yml

```diff
@@ -30,4 +30,9 @@
 # available at http://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
+  new_rent: 'New rent created'
```

Comment:
You can nest translations!
```yml
es:
  rent_mailer:
    new_rent_notification:
      created_at: 'Created at'
      ....
```
Using this naming convention, Rails will know where to look every translation and you can do `t('.created_at')` in the view file, without specifying the nested keys